test/clj/milia/api/io_test.clj

```diff
@@ -184,6 +185,21 @@
                                       url
                                       appended-options) => :response))))
 
+       (fact "should add access-token if access-token exists while auth-token
+              and temp-token do not exist. "
+             (binding [*credentials* (assoc {} :access-token access-token-map)]
```

Comment:
This will result to `*credentials*` having `{:access-token {:access-token "access token"}}`. We'll have 2 keys that are the same. Is it possible for us to assign `access-token-map` to `*credentials*`?

Reply:
Will change this